lib/package.json

```diff
@@ -31,17 +34,19 @@
     "@cypress/vite-dev-server": "^5.0.5",
     "@cypress/vue": "^5.0.5",
     "@vitejs/plugin-vue": "^4.2.1",
+    "@vitest/ui": "^0.31.0",
     "@vue/babel-preset-app": "^5.0.8",
```

Comment:
I think we don't need this dep, neither `@babel/core` or any Babel related as it was only used by Jest